source/_integrations/roborock.markdown

```diff
@@ -32,127 +32,246 @@ ha_platforms:
+  - **Description**: This disables the buttons on the vacuum. Nothing happens when the buttons are pushed.
 
-Do not disturb - This enables _Do not disturb_ during the time frame you have set in the app or on the time entity. When _Do not disturb_ is enabled, the vacuum does not run or speak.
+- **Status indicator light**
+  - **Description**: This is the LED on the top of your vacuum. The color changes depending on the status of your vacuum.
 
-### Number
+- **Do not disturb**
+  - **Description**: This enables _Do not disturb_ during the time frame you have set in the app or on the time entity. When _Do not disturb_ is enabled, the vacuum does not run or speak.
 
-Volume - This allows you to control the volume of the robot's voice. For example, when it states "Starting cleaning". This allows you to set the volume to 0%, while the app limits it to 20%.
+#### Number
 
-### Button
+- **Volume**
+  - **Description**: This allows you to control the volume of the robot's voice. For example, when it states "Starting cleaning". This allows you to set the volume to 0%, while the app limits it to 20%.
+
+#### Button
 
 There are currently four buttons that allow you to reset the various maintenance items on your vacuum. Pressing the button cannot be undone. For this reason, the buttons are disabled by default to make sure they are not pressed unintentionally.
 
-Reset sensor consumable - The sensors on your vacuum are expected to be cleaned after 30 hours of use.
+- **Reset sensor consumable**
+  - **Description**: The sensors on your vacuum are expected to be cleaned after 30 hours of use.
+
+- **Reset side brush consumable**
+  - **Description**: The side brush is expected to be replaced every 200 hours.
 
-Reset side brush consumable - The side brush is expected to be replaced every 200 hours.
+- **Reset main brush consumable**
+  - **Description**: The main brush/ roller is expected to be replaced every 300 hours.
 
-Reset main brush consumable - The main brush/ roller is expected to be replaced every 300 hours.
+- **Reset air filter**
+  - **Description**: The air filter is expected to be replaced every 150 hours.
 
-Reset air filter - The air filter is expected to be replaced every 150 hours.
+#### Actions
 
-### Actions
+##### Action Set Vacuum Goto Position
 
-#### Action `roborock.set_vacuum_goto_position`
+The `roborock.set_vacuum_goto_position` action will set the vacuum to go to
+the specified coordinates.
 
-Go the specified coordinates.
 
-- **Data attribute**: `entity_id`
-  - **Description**: Only act on a specific robot.
-  - **Optional**: No.
-- **Data attribute**: `x_coord`
-  - **Description**: X-coordinate, integer value. The dock is located at x-coordinate 25500.
-  - **Optional**: No.
-- **Data attribute**: `y_coord`
-  - **Description**: Y-coordinate, integer value. The dock is located at y-coordinate 25500.
-  - **Optional**: No.
+| Data attribute   | Optional | Description                                          |
```

Comment:
so, about this... I like the original version here better. I know the example in the dev docs uses a table. Looking at this, I think the example should be changed. Can you revert this table and the one below?

Reply:
https://github.com/home-assistant/developers.home-assistant/pull/2602

Reply:
Done.